extensions/api/v1alpha1/sandboxclaim_types.go

```diff
@@ -37,6 +21,43 @@ import (
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 // Important: Run "make" to regenerate code after modifying this file
 
+const (
+	// ClaimExpiredReason is the reason used in conditions/events when a claim expires.
+	ClaimExpiredReason = "ClaimExpired"
+)
+
+// ShutdownPolicy describes the policy for shutting down the underlying Sandbox when the SandboxClaim expires.
+// +kubebuilder:validation:Enum=Delete;Retain
+type ShutdownPolicy string
+
+const (
+	// ShutdownPolicyDelete deletes the SandboxClaim (and cascadingly the Sandbox) when expired.
+	ShutdownPolicyDelete ShutdownPolicy = "Delete"
+
+	// ShutdownPolicyRetain keeps the SandboxClaim when expired (Status will show Expired).
+	// The underlying SandboxClaim resources (Pod, Service) are deleted to save resources,
+	// but the SandboxClaim object itself remains.
+	ShutdownPolicyRetain ShutdownPolicy = "Retain"
+)
+
+// Lifecycle defines the lifecycle management for the SandboxClaim.
+type Lifecycle struct {
+	// ShutdownTime is the absolute time when the SandboxClaim expires.
+	// This time governs the lifecycle of the claim. It is propagated to the
+	// underlying Sandbox, overriding any expiration logic defined there.
+	//
+	// Updates are allowed and if this field is updated to be omitted (nil),
```

Comment:
I wonder if it's necessary to explicitly call out that this field can be updated. `ShutdownPolicy` can be updated too. What about other fields?